src/sagemaker/image_uri_config/pytorch.json

```diff
@@ -26,16 +25,22 @@
                     "eu-central-1": "763104351884",
                     "eu-north-1": "763104351884",
```

Comment:
All these regions, apart from `ap-northeast-1`, `ap-northeast-2`, `eu-west-1`, `us-east-1`, `us-east-2`, and `us-west-2`, cannot create EI inference endpoints because EI is unsupported in those regions.

The list of regions for PT 1.5.1 is correct, and the list of regions for PT 1.3.1 needs to be updated to match that.

Reference: https://aws.amazon.com/machine-learning/elastic-inference/pricing/

Reply:
Thanks for clarifying! As discussed, this does lead to a change in customer experience whereby if these regions are non existent, we won't be able to construct the image URIs for the customer properly, and it will diverge from the current error thrown in this scenario. Since we aren't supporting EIA in these regions prior to this, I think this is a correct bug fix in that regard; and we should in general think of introducing a framework which checks for feature/region compatibility and alerts users accordingly, which will allow for easier and more seamless deprecation